demoScripts/vmssQuickstart.md

```diff
@@ -0,0 +1,254 @@
+                      "resourceGroup": "myResourceGroup"
+                    }
+                  }
+                }
+              ],
+              "primary": true
+            }
+          }
+        ]
+      },
+      "osProfile": {
+        "adminUsername": "azureuser",
+        "allowExtensionOperations": true,
+        "computerNamePrefix": "mysca2132",
+        "linuxConfiguration": {
+          "disablePasswordAuthentication": true,
+          "enableVMAgentPlatformUpdates": false,
+          "provisionVMAgent": true,
+          "ssh": {
+            "publicKeys": [
+              {
+                "keyData": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCvR1+fGFuVMWS2bAY0SgW4E9QzLZ77ETdbCBUVF46eAyL8JWsLynX214hNSK16l4UYZyC3E6jea5qw2rGHPP4eMp7iif50xqd6qGICS428mqc9Gz29J0LFanM7XpHwLnBiJ6hmKvqvHB5tsGKh44MddW0wv+KiiEHIV1ZdSSvBRJ5MMQhqZoUiqlChHourOhaZxvw2dpJhRCvAEKw1s5RoeoLJAdZ6Qr53ERSkJr3BF7uAoNlGx6gatBVkjV+w9CZXN/YN62b1QQiGnk5/BIXNqEIsyxsa84+GbyieRIN/wYjSEV7ASRxSj60qV7RPexvAI+4JGa9UELYMQDrBElgL",
+                "path": "/home/azureuser/.ssh/authorized_keys"
+              }
+            ]
+          }
+        },
+        "requireGuestProvisionSignal": true,
+        "secrets": []
+      },
+      "storageProfile": {
+        "imageReference": {
+          "offer": "UbuntuServer",
+          "publisher": "Canonical",
+          "sku": "18.04-LTS",
+          "version": "latest"
+        },
+        "osDisk": {
+          "caching": "ReadWrite",
+          "createOption": "FromImage",
+          "diskSizeGB": 30,
+          "managedDisk": {
+            "storageAccountType": "Premium_LRS"
+          },
+          "osType": "Linux"
+        }
+      }
+    }
+  }
+}
+```
+
+## Allow traffic to application
+When the scale set was created, an Azure load balancer was automatically deployed. The load balancer distributes traffic to the VM instances in the scale set. To allow traffic to reach the sample web application, create a load balancer rule with [az network lb rule create](/cli/azure/network/lb/rule). The following example creates a rule named *myLoadBalancerRuleWeb*:
+
+```azurecli-interactive
+az network lb rule create \
+  --resource-group $RESOURCE_GROUP_NAME \
+  --name $LOAD_BALANCER_RULE_NAME \
+  --lb-name $LOAD_BALANCER_NAME \
+  --backend-pool-name $BACKEND_POOL_NAME \
+  --backend-port 80 \
+  --frontend-ip-name $FRONT_END_IP_NAME \
+  --frontend-port 80 \
+  --protocol tcp
+```
+
+## Test your scale set
+To see your scale set in action, access the sample web application in a web browser. Obtain the public IP address of your load balancer with [az network public-ip show](/cli/azure/network/public-ip). The following example obtains the IP address for *myScaleSetLBPublicIP* created as part of the scale set:
+
+```azurecli-interactive
+az network public-ip show \
```

Comment:
No results block. As with another review we should be putting the IP Address in an environment variable for reuse.